package.json

```diff
@@ -120,12 +124,11 @@
     "get-port": "^3.2.0",
     "glob": "^7.1.6",
     "graphql": "0.13.2",
-    "int64-buffer": "^0.1.9",
+    "gzip-js": "^0.3.2",
     "jszip": "^3.5.0",
     "knex": "^2.4.2",
     "mkdirp": "^0.5.1",
-    "mocha": "8",
-    "msgpack-lite": "^0.1.26",
+    "mocha": "^10.2.0",
```

Comment:
This breaks compatibility with Node 12.

Reply:
After looking through the code we didn't notice anything that requires a new version of Mocha. There aren't any notes as to why Mocha was upgraded. For that reason we'll try downgrading the version and checking if the tests still run.

Reply:
I just ran `npm install msgpack-lite --save`, and this dependency got upgraded.